src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -565,6 +565,11 @@ internal override void ValidateOptions(ArrayBuilder<Diagnostic> builder)
                     builder.Add(Diagnostic.Create(MessageProvider.Instance, (int)ErrorCode.ERR_MutuallyExclusiveOptions, nameof(CryptoPublicKey), nameof(CryptoKeyContainer)));
                 }
             }
+
+            if (PublicSign && DelaySign.HasValue)
```

Comment:
Is this a typo? What if DelaySign has value "false"? If command line has "/delaysign-" parameter?